NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -291,10 +292,66 @@ public void run() {
 			catch (SQLException e) {
 				
 			}
+			//old permission table is no longer used, so delete it
```

Comment:
This seems like a bad idea.

Have it as a separate update, but don't modify a prior database migration that is already "in the field". A migration should be effectively "code locked" as soon as its deployed, only actual functional fixes but no functionality alterations. 

If dropping the table is important but it stands alone from ver 12, make it its own version. If you only want this to run for people who are updating from <v10 to v12... I'd recommending flagging that that state exists before v11 update, and running this little piece of code in its own block after v11 update completed.

Might just be pedantic but generally its better not to modify existing migrations due to secondary effects.